src/js/components/KillTaskModal.js

```diff
@@ -43,46 +44,22 @@ class KillTaskModal extends mixin(StoreMixin) {
     this.props.onSuccess();
   }
 
-  getContentHeader(selectedItems, selectedItemsLength) {
-    let {action} = this.props;
-    let headerContent = ` ${selectedItemsLength} Tasks`;
-    if (selectedItemsLength === 1) {
-      headerContent = ` ${selectedItems[0]}`;
-    }
-
-    return (
-      <h2 className="flush-top text-align-center" key="confirmHeader">
-        {`Are you sure you want to ${ACTION_DISPLAY_NAMES[action]} ${headerContent}?`}
-      </h2>
-    );
-  }
-
-  getConfirmTextBody(selectedItems, selectedItemsLength) {
-    let {action} = this.props;
-    let bodyText;
-
-    if (selectedItemsLength === 1) {
-      bodyText = selectedItems[0];
-    } else {
-      bodyText = 'these tasks';
-    }
-
-    return (
-      <span key="confirmText">
-        You are about to {ACTION_DISPLAY_NAMES[action]} {bodyText}.
-      </span>
-    );
-  }
-
   getModalContents() {
-    let {selectedItems} = this.props;
-    let selectedItemsLength = selectedItems.length;
+    let selectedItemsLength = this.props.selectedItems.length;
+    let action = ACTION_DISPLAY_NAMES[this.props.action] || '';
+    let taskCountContent = ` ${selectedItemsLength} ${StringUtil.pluralize('Task', selectedItemsLength)}`;
```

Comment:
Unless this is intentional, you could remove the space, since this is already spaced on line 59